osquery/tables/system/darwin/acpi_tables.cpp

```diff
@@ -51,13 +52,16 @@ QueryData genACPITables(QueryContext& context) {
     return {};
   }
 
-  CFTypeRef table = IORegistryEntryCreateCFProperty(service, CFSTR("ACPI Tables"), kCFAllocatorDefault, 0);
+  CFTypeRef table = IORegistryEntryCreateCFProperty(
+      service, CFSTR(kIOACPIPropertyName_), kCFAllocatorDefault, 0);
   if (table == nullptr) {
+    IOObjectRelease(service);
     return {};
   }
 
   CFDictionaryApplyFunction((CFDictionaryRef)table, genACPITable, &results);
 
+  CFRelease(table);
```

Comment:
Correct, looks like the previous implementation was not releasing properly. Perhaps we should release block on this PR.